client/packages/invoices/src/Returns/api/operations.graphql

```diff
@@ -414,6 +423,9 @@ mutation updateCustomerReturnLines(
     ... on InvoiceNode {
       __typename
       id
+      originalShipment {
+        id
+      }
```

Comment:
This adding to graphql (and for updateSupplierReturnLines) is for type compatibility.

I wonder if it is possible to not return this type and have type compatibility, but I couldn't see how to.